src/Console/InstallCommand.php

```diff
@@ -424,21 +411,52 @@ protected function ensureApplicationIsTeamCompatible()
     }
 
     /**
-     * Install the Jetstream service providers in the application configuration file.
+     * Install the service provider in the application configuration file.
      *
+     * @param  string $after
+     * @param  string $name
      * @return void
      */
-    protected function installJetstreamServiceProvider()
+    protected function installServiceProviderAfter($after, $name)
     {
-        if (! Str::contains($appConfig = file_get_contents(config_path('app.php')), 'App\\Providers\\JetstreamServiceProvider::class')) {
+        if (! Str::contains($appConfig = file_get_contents(config_path('app.php')), 'App\\Providers\\'.$name.'::class')) {
             file_put_contents(config_path('app.php'), str_replace(
-                "App\\Providers\FortifyServiceProvider::class,",
-                "App\\Providers\FortifyServiceProvider::class,".PHP_EOL."        App\Providers\JetstreamServiceProvider::class,",
+                'App\\Providers\\'.$after.'::class,',
+                'App\\Providers\\'.$after.'::class,'.PHP_EOL.'        App\\Providers\\'.$name.'::class,',
                 $appConfig
             ));
         }
     }
 
+    /**
+     * Install the middleware to a group in the application Http Kernel.
+     *
+     * @param  string $after
+     * @param  string $name
+     * @param  string $group
+     * @return void
+     */
+    protected function installMiddlewareAfter($after, $name, $group = 'web')
+    {
+        $httpKernel = file_get_contents(app_path('Http/Kernel.php'));
+        $middlewareGroups = Str::before(Str::after($httpKernel, '$middlewareGroups = ['.PHP_EOL), '];'.PHP_EOL);
```

Comment:
Particularly here. On Windows I'm not sure this Str::after call will return what you want.

Reply:
Yeah, I think you're right.. I totally forgot about Windows doing things like this differently.

Let me see if I can adjust this.